poetry/core/packages/package.py

```diff
@@ -88,7 +88,8 @@ def __init__(
         self.documentation_url = None
         self.keywords = []
         self._license = None
-        self.readme = None
+        self.readme = ()
+        self.readme_type = None
```

Comment:
Since the description is a sum of READMEs, I'd probably call this the `description_type` instead.